.travis.yml

```diff
@@ -16,14 +16,15 @@ before_install:
   - ./miniconda.sh -b -p ./miniconda
   - export PATH=`pwd`/miniconda/bin:$PATH
   - conda update --yes conda
-  - conda install --yes psutil
-  - conda install --yes -c conda-forge cartopy pandas PIL rasterio six pytest
-  - pip install mercantile coveralls pytest-cov
+  - conda config --add channels conda-forge --force
+  - conda create --yes -n TEST python=$TRAVIS_PYTHON_VERSION --file requirements.txt --file requirements-dev.txt
+  - source activate TEST
+  - pip install coveralls
 
 install:
   - python setup.py install
 
-script: 
```

Comment:
There is some noise due to blank space in this PR. Let me know if that is a problem and I can re-send it without them.